external/thorntail-mp-tck/README.md

```diff
@@ -0,0 +1,31 @@
+# External Thorntail Microprofile TCK Tests
+
+Thorntail Microprofile TCK tests are a part of the external third-party application tests that help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/[Issue #172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests. Thorntail-mp-tck test material is pulled from the [thorntail](https://github.com/thorntail/thorntail) repository.
+
+## Running External Thorntail-mp-tck tests locally
+
+To run AQA tests locally using thorntail-mp-tck, you follow the same pattern:
+
+0. Ensure your test machine is set up with [test prereqs](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/Prerequisites.md). For external tests, you do need Docker installed.
+
+1. Download/unpack the SDK you want to on your test machine
```

Comment:
Please update to Download/unpack the SDK that you want to test on your test machine

Reply:
Thanks @sophia-guo, made the required changes.
Kindly review.